changelogs/unreleased/gh-8793-rework-sql-defaults.md

```diff
@@ -0,0 +1,3 @@
+## feature/sql
+
+* Space no longer lose default value after `ALTER TABLE ADD COLUMN` (gh-8793).
```

Comment:
Minor: According to the description it's rather bugfix that a feature.

Reply:
Fixed. I added another changelog file.